doc/services/device_mgmt/smp_groups/smp_group_2.rst

```diff
@@ -117,7 +117,7 @@ Statistics: list of groups
 
 The command is used to obtain list of groups of statistics that are gathered
 on a device. This is a list of names as given to groups with
-``STATS_INIT_AND_REG`` macro or ``stats_init_and_reg(...)`` function calls,
+``STATS_INIT_AND_REG`` macro or :c:func:`stats_init_and_reg` function calls,
```

Comment:
Shouldn't the `STAT_INIT_AND_REG` receive `:c:macro:` prefix?

Reply:
Fixed, will close and submit a new PR since the documentation rebuild does not seem to be getting uploaded